server/datastore/mysql/software_titles.go

```diff
@@ -219,8 +220,14 @@ func (ds *Datastore) ListSoftwareTitles(
 		if i, ok := titleIndex[p.TitleID]; ok {
 			if softwareList[i].AppStoreApp != nil {
 				softwareList[i].AppStoreApp.AutomaticInstallPolicies = append(softwareList[i].AppStoreApp.AutomaticInstallPolicies, p)
-			} else {
+			} else if softwareList[i].SoftwarePackage != nil {
```

Comment:
This is not the fix. I'm adding this in case it happens in the future. The fix is in the use of `teamID` in `getPoliciesBySoftwareTitleIDs`.